### ISSUE TITLE
Error running generate-openapi-spec script — ACA-Py fails to start in image

### ISSUE THREAD

**Issue by swcurran** [hyperledger/aries-cloudagent-python#2798]
Tried to generate the open-api file using the provided script using the command: ``

The expected Admin API web interface does not come up. I tweaked the script to not run the docker image in daemon mode (removed `-d` option from docker startup), and got this error message:

```
##########################################################################################
#
#  Starting ACA-py docker image with command: 
         docker run --rm --name aries-cloudagent-run_KUasT9ETe1QgzFGE -p 8307:8307 -p 8305:8305 aries-cloudagent-run start -e http --inbound-transport http 0.0.0.0 8307 --outbound-transport http --admin 0.0.0.0 8305 --admin-insecure-mode --log-level info --auto-provision --wallet-type askar --wallet-name gen-openapi --wallet-key gen-openapi-key --multitenant --multitenant-admin --jwt-secret test --no-ledger --log-file /usr/src/app/logs/agent.log
#
##########################################################################################

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/src/app/aries_cloudagent/__main__.py", line 69, in script_main
    main(sys.argv)
  File "/usr/src/app/aries_cloudagent/__main__.py", line 75, in main
    run(args)
  File "/usr/src/app/aries_cloudagent/__main__.py", line 64, in run
    run_command(command, args)
  File "/usr/src/app/aries_cloudagent/commands/__init__.py", line 38, in run_command
    module.execute(argv)
  File "/usr/src/app/aries_cloudagent/commands/start.py", line 50, in execute
    common_config(settings)
  File "/usr/src/app/aries_cloudagent/config/util.py", line 19, in common_config
    LoggingConfigurator.configure(
  File "/usr/src/app/aries_cloudagent/config/logging.py", line 183, in configure
    fileConfig(
  File "/usr/src/app/aries_cloudagent/config/logging.py", line 130, in fileConfig
    handlers = _install_handlers(cp, formatters)
  File "/usr/local/lib/python3.9/logging/config.py", line 145, in _install_handlers
    h = klass(*args, **kwargs)
  File "/usr/src/app/aries_cloudagent/config/logging.py", line 376, in __init__
    BaseRotatingHandler.__init__(
  File "/usr/local/lib/python3.9/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1146, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1175, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding,
FileNotFoundError: [Errno 2] No such file or directory: '/usr/src/app/logs/agent.log'
**** FAIL - ACA-Py failed to start, exiting. ****
```

**Comment by swcurran** [comment]
I was able to fix this be removing the `—log_file` item from the list of ACA-Py startup options.

@dbluhm — since you were the last to touch this, I’ll flag you, but no big deal.  It definitely did as expected when run with the tweak.  I included it in the 0.12.0rc1 PR.

**Comment by swcurran** [comment]
Fixed in 0.12.0rc1 release PR.
